explorer/websockethandlers.go

```diff
@@ -250,15 +250,17 @@ func (exp *explorerUI) RootWebsocket(w http.ResponseWriter, r *http.Request) {
 				}
 
 				if !sig.IsValid() {
-					break loop
+					log.Errorf("invalid signal to send: %s / %d", sig.Signal.String(), int(sig.Signal))
+					continue
```

Comment:
No longer disconnect, just log the error and wait for more events to send to the client.